tests/health_checks/NDv4/run-health-checks.sh

```diff
@@ -7,12 +7,44 @@
 #Verify if an argument was passed to the script. Test if the argument is
 #a directory that contains all the expected health checks. If no arguments
 #are passed use the default directory.
```

Comment:
Please add a check of number of arguments, then echo script usage if it's incorrect.

Reply:
Added this